test/test_topic.py

```diff
@@ -402,31 +384,31 @@ def test_msg(self):
 
     def test_msg_cb(self):
         result = self.topic_msg_cb.tick()
-        msg = rospy.wait_for_message('/topic_cb', String)
+        msg = rospy.wait_for_message('topic_cb', String)
```

Comment:
whats the reason for the wait_for_message here vs the calling the callback directly?